### PR TITLE
Add Aeotec ZW111-C EU firmware version 2.4

### PR DESCRIPTION
**Pull request by larstobi**
Add firmware upgrade for Aeotec Nano Dimmer ZW111-C EU version 2.04.

Firmware page:
https://aeotec.freshdesk.com/support/solutions/articles/6000198943-how-to-update-nano-dimmer-z-wave-firmware-

### CODE REVIEW

**Review by AlCalzone — COMMENTED** [review]
@Aeotec-ccheng can you review this please?

**Comment by larstobi** [comment]
Is it the integrity of the zip-file or the .exe file that we want as the sha256 sum?

**Comment by AlCalzone** [comment]
Z-Wave JS doesn't support zip files for firmware updates currently, so we'd need a link to the file contained in the ZIP.
If you run the URL through `npx @zwave-js/firmware-integrity <url>`, it will give you the correct integrity - if the file is in a supported format.

**Comment by larstobi** [comment]
> Z-Wave JS doesn't support zip files for firmware updates currently, so we'd need a link to the file contained in the ZIP.

Aha. Since the file must be hosted by the manufacturer, and since the manufacturer has posted the file as a .zip file, then we cannot get this firmware to work with firmware-updates.

**Comment by Aeotec-ccheng** [comment]
I'll see if i can grab the OTZ files, those can be used instead of the zip files we supply on that article. Hopefully should be able to get these files earlier next week after christmas (may not hear back until after new years).

**Comment by Aeotec-ccheng** [comment]
https://aeotec.freshdesk.com/a/solutions/articles/6000198943 OTZ files posted in this link here @larstobi 

Specifically for EU Frequency, this should be the link here: https://aeotec.freshdesk.com/helpdesk/attachments/6189873103

Remember to generate a new SHA key.

**Comment by larstobi** [comment]
Excellent! Thanks, will update the PR.

**Comment by Aeotec-ccheng** [comment]
@AlCalzone this one should be good now. I'll make some for AU and US frequency once i get back to office on Jan 2nd.

File: firmwares/aeotec/ZW111-C.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		{
+			"$if": "firmwareVersion >= 1.0 && firmwareVersion < 2.4",
+			"version": "2.4",
+			"channel": "stable",
+			"changelog": "When using Action Button to factory reset (long press 20 seconds), all parameters (including parameter 120, 121, 128, 129, 130, 131, 132) will be reset to factory default settings.",
+			"files": [
+				{
+					"target": 0,
+					"url": "https://aeotec.freshdesk.com/helpdesk/attachments/6189873103",
+					"integrity": "sha256:165c1a652eeaf96c0d28370cd8480bf9bb3206dc9321e3d663b0de9d199a3082"
```
Comment (AlCalzone) [inline firmwares/aeotec/ZW111-C.json:21]:
```suggestion
					"integrity": "sha256:aba71a9c6f2065b8fc5e683b81ebeea7cfbb1f66342be969d48eb5b671479b7e"
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Oh, I just saw the PR by @Aeotec-ccheng includes this one.

**Pull request closed without merging**